hikari/impl/bot.py

```diff
@@ -357,17 +355,22 @@ def _check_if_alive(self) -> None:
         if not self._is_alive:
             raise errors.ComponentStateConflictError("bot is not running so it cannot be interacted with")
 
+    def _get_closing_event(self) -> asyncio.Event:
+        if not self._closing_event:
+            self._closing_event = asyncio.Event()
+
+        return self._closing_event
```

Comment:
This might lead to some unexpected cases where we unset `_closing_event` but this call will set it again

Reply:
That function is no more 🦀